src/scripts/main.js

```diff
@@ -1,3 +1,317 @@
+  const ageInput = document.createElement('input');
+
+  ageLabel.textContent = 'Age:';
+  ageInput.type = 'number';
+  ageInput.name = 'age';
+  ageInput.dataset.qa = 'age';
+  ageInput.required = true;
+  ageInput.setAttribute('data-qa', 'age');
+  ageLabel.appendChild(ageInput);
+
+  const salaryLabel = document.createElement('label');
+  const salaryInput = document.createElement('input');
+
+  salaryLabel.textContent = 'Salary:';
+  salaryInput.type = 'number';
+  salaryInput.name = 'salary';
+  salaryInput.dataset.qa = 'salary';
+  salaryInput.required = true;
+  salaryInput.setAttribute('data-qa', 'salary');
+  salaryLabel.appendChild(salaryInput);
+
+  const countryLabel = document.createElement('label');
+  const countrySelect = document.createElement('select');
+
+  countryLabel.textContent = 'Office:';
+  countrySelect.name = 'office';
+  countrySelect.dataset.qa = 'office';
+  countrySelect.required = true;
+  countrySelect.setAttribute('data-qa', 'office');
+
+  const countries = [
+    'Tokyo',
+    'Singapore',
+    'London',
+    'New York',
+    'Edinburgh',
+    'San Francisco',
+  ];
+
+  countries.forEach((country) => {
+    const option = document.createElement('option');
+
+    option.value = country;
+    option.textContent = country;
+    countrySelect.appendChild(option);
+  });
+
+  countryLabel.appendChild(countrySelect);
+
+  const submitButton = document.createElement('button');
+
+  submitButton.type = 'button';
+  submitButton.name = 'submit';
+  submitButton.textContent = 'Save to table';
+
+  form.appendChild(nameLabel);
+  form.appendChild(positionLabel);
+  form.appendChild(countryLabel);
+  form.appendChild(ageLabel);
+  form.appendChild(salaryLabel);
+  form.appendChild(submitButton);
+
+  document.body.appendChild(form);
+
+  const notification = document.createElement('div');
+
+  function showNotification(message, isSuccess) {
+    clearNotifications();
+
+    notification.id = 'notification';
+    notification.setAttribute('data-qa', 'notification');
+    notification.style.padding = '8px';
+    form.appendChild(notification);
+
+    notification.textContent = message;
+    notification.className = isSuccess ? 'success' : 'error';
+    notification.style.color = isSuccess ? 'green' : '#e25644';
+    notification.style.display = 'block';
+  }
+
+  const clearNotifications = () => {
+    notification.innerHTML = '';
+  };
+
+  function validateForm() {
+    let valid = true;
+    const nameVal = nameInput.value.trim();
+    const ageVal = ageInput.value;
+    const positVal = positionInput.value;
+
+    if (nameVal.length < 4) {
+      showNotification('Name must have at least 4 letters.', false);
+      valid = false;
+    }
+
+    if (ageVal < 18 || ageVal > 90) {
+      showNotification('Age must be between 18 and 90.', false);
+      valid = false;
+    }
+
+    if (positVal < 1) {
+      showNotification('Position cant` be empty', false);
+      valid = false;
+    }
```

Comment:
Validation issue: The condition `positVal < 1` is incorrect for validating the position field, as it checks for a numeric value. Instead, you should check if the trimmed string length is less than a certain threshold (e.g., `positVal.trim().length < 1`).